LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2617,6 +2617,43 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
                         c.Add(new Control("Chipset Fan", 3));
                         break;
 
+                    case Model.Z690_Extreme:
+                        v.Add(new Voltage("Vcore", 0, 10, 10));
+                        v.Add(new Voltage("+5V", 1, 20, 10));
+                        v.Add(new Voltage("AVCC", 2, 34, 34));
+                        v.Add(new Voltage("+3.3V", 3, 34, 34));
+                        v.Add(new Voltage("+12V", 4, 110, 10));
+                        v.Add(new Voltage("CPU Input Voltage", 5, 1, 1));
```

Comment:
I noticed that CPU VCCIN_AUX was renamed to CPU Input Auxiliary [in this commit](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/commit/beb3d35a1c1e6653bdc0ef1e2b91c11894332f20#diff-ab954123283065b240cec0d9311bd2feffade5803136f44cfe7d89242a81373aR1566). Would you rather have that as the name for consistency, or would you prefer what you previously suggested to me?

Reply:
CPU Input Auxiliary is fine, I generally try to avoid adding "Voltage" for names given it's already a voltage.